src/sv_ents.c

```diff
@@ -730,6 +762,11 @@ static void SV_WritePlayersToClient (client_t *client, client_frame_t *frame, by
 
 		if (pflags & PF_WEAPONFRAME)
 			MSG_WriteByte (msg, ent->v->weaponframe);
+
+#ifdef FTE_PEXT_TRANS
+		if (pflags & PF_TRANS_Z)
+			MSG_WriteByte (msg, ent->xv.alpha ? bound(1, (byte)(ent->xv.alpha * 254.0f), 254) : 255);
```

Comment:
Should actually check >0 && < 1 like the other places, and default to zero I think.